### PR TITLE
Censorship conditionals

### PR DESCRIPTION
**Pull request by Norgus**
I think this is ready to go back into the main branch.

The portions of script which were censored in our latest script VS the original MG PC release are now switched depending on the censorship (voice matching) flag.

### CODE REVIEW

File: Update/omake_01.txt [hunk 1]
```diff
@@ -1594,6 +1488,8 @@ void main()
 		   NULL, " So half of us think it was a curse, and half of us think it was people?\"", GetGlobalFlag(GLinemodeSp));
 	if (GetGlobalFlag(GADVMode)) { ClearMessage(); } else { OutputLineAll(NULL, "\n", Line_ContinueAfterTyping); }
 
+	if(GetGlobalFlag(GCensor) == 0){ModCallScriptSection("zomake_01_vm00_n01","dialog004");}
```
Comment (enumag) [inline Update/omake_01.txt]: You sure about this? We generally use the GCensor for places where PS3 and original have different dialogue. But I don't think we ever used it to simply cut out something.

Comment (Norgus) [reply Update/omake_01.txt]: I think I was being too sensitive to the idea of having unvoiced lines here. I don't mind putting it back.

**Comment by enumag** [comment]
Wow, that's a lot of changes. Does this require any other additional resources? Voices, sprites, backgrounds? Also is it complete or are you still working on it?

**Comment by Norgus** [comment]
Well, I completed the work that I started the branch for - converting in-line censored text to flag-switched conditional script by comparing the non-modded script with our most recent one.

I then started re-comparing our script with the Sui one, which isn't resulting in any major changes so far. I feel it would make as much sense to continue doing this part on master.

**Pull request merged**